troubleshooting/faq.md

```diff
@@ -161,6 +162,18 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## How do I update my npm Enterprise license?
+
+You can update npm Enterprise license by following the steps below:
+
+1. Run `npme update-license` on your npm Enterprise server.
+
+2. It will prompt you for billing email, enter your registered billing email.
+
+3. Then it will prompt you for license key, enter your license key.
+
+You can get license key by purchasing npm Enterprise license from [here](https://www.npmjs.com/enterprise/license)
```

Comment:
Minor change in wording/grammar: "You can get a license key by purchasing an npm Enterprise license on our [license page](https://www.npmjs.com/enterprise/license)."